containers/Dockerfile.builder-base

```diff
@@ -2,8 +2,6 @@
 FROM registry.fedoraproject.org/fedora-minimal:30 as base
 ARG RUBY_VERSION="2.6"
 ARG NODEJS_VERSION="11"
-ENV FOREMAN_FQDN=foreman.example.com
-ENV FOREMAN_DOMAIN=example.com
```

Comment:
why? this will fail the production image (its needed to build and to run) if you don't include it here.

Reply:
Just wrong conflict resolution (had it in different image). Fixed